Clue/Classes/Modules/UserInteractionModule.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  UserInteractionModule.swift
+//  Clue
+//
+//  Created by Andrea Prearo on 5/8/17.
+//  Copyright © 2017 Ahmed Sulaiman. All rights reserved.
+//
+
+import Foundation
+
+let UserInteractionTouchBeganEvent = "Began"
+let UserInteractionTouchMovedEvent = "Moved"
+let UserInteractionTouchEndedEvent = "Ended"
```

Comment:
What do you think about changing those constants into enum type with `String` as a raw type?
Something like this:
```swift
enum UserInteractionEventType: String {
    case began = "Began"
    case moved = "Moved"
    case ended = "Ended"
}
```